VideoWeb/VideoWeb/ClientApp/src/app/services/conference/participant.service.ts

```diff
@@ -154,6 +153,11 @@ export class ParticipantService {
             this._nonEndpointParticipants = [];
             this._virtualMeetingRooms = [];
 
+            if (!conference) {
+                this.logger.warn(`${this.loggerPrefix} no conference loaded; skipping initialisation.`);
+                return;
+            }
+
```

Comment:
Same as in the other service it would be good to test it doesn't proceed when falsy